tests/ui_tests/conftest.py

```diff
@@ -47,13 +50,35 @@ def additional_adcm_init_config(request) -> dict:
 
 
 @pytest.fixture(scope="session")
-def web_driver(browser):
+def downloads_directory(tmpdir_factory: TempdirFactory) -> py.path.local:
+    """Folder in which browser downloads will be stored"""
+    downloads_dirname = 'browser-downloads'
+    return tmpdir_factory.mktemp(downloads_dirname)
+
+
+@pytest.fixture()
+def clean_downloads_fs(request: SubRequest, downloads_directory: py.path.local):
+    """Clean downloads directory before use"""
+    for item in downloads_directory.listdir():
+        item.remove()
```

Comment:
why not `tmpdir`  fixture?

Reply:
because we need to specify download directory in `ADCMTest` constructor and fixture that initializes it is session-scoped